docusaurus.config.js

```diff
@@ -93,14 +93,26 @@ const config = {
             label: 'Tutorials',
             position: 'left', // or 'right', depending on where you want it in the navbar
           },
+          {
+            to: 'https://api.playcanvas.com',  // This should match the route of your Tutorials page
```

Comment:
Shouldn't this be an `href` to show the external link arrow?

Reply:
Ahh yeah, good catch! Fixed.